src/main/java/org/alliancegenome/curation_api/model/ingest/json/dto/ExperimentalConditionDTO.java

```diff
@@ -13,7 +15,8 @@
     private String conditionQuantity;
     private String anatomicalOntologyId;
     private String geneOntologyId;
-    private String ncbitaxonId;
+    @JsonProperty("NCBITaxonId")
```

Comment:
I don't think this works for every mod? This works only for ZFIN and ZFIN has produced a new file so that we don't need the json property? @cmpich ?

Reply:
Only ZFIN is submitting that field, but it is in the capitalised form in the frozen JSON schema and the latest file was failing to load.